### ISSUE TITLE
can't build jiffy

### ISSUE THREAD

**Issue by alexshavelev** [ninenines/erlang.mk#816]
after updating erlang.mk to the latest version can't build jiffy (with use FULL=1 flag)

first
```
DEP    jiffy (0.13.0)
erlang.mk:4949: warning: ignoring old recipe for target `/home/jenkins/workspace/myapp/deps/eovsdb'
{"init terminating in do_boot",{badarg,[{re,replace,["priv/jiffy.so","(\414)(?![(])(\w*)","\1(\2)",[global]],[{file,"re.erl"},{line,357}]},{erl_eval,do_apply,6,[{file,"erl_eval.erl"},{line,670}]},{erl_eval,expr,5,[{file,"erl_eval.erl"},{line,438}]},{erl_eval,exprs,5,[{file,"erl_eval.erl"},{line,122}]},{erl_eval,expr_list,6,[{file,"erl_eval.erl"},{line,878}]},{erl_eval,expr,5,[{file,"erl_eval.erl"},{line,236}]},{erl_eval,eval_lc1,6,[{file,"erl_eval.erl"},{line,696}]},{erl_eval,eval_generate,7,[{file,"erl_eval.erl"},{line,725}]}]}}
init terminating in do_boot ()
```

it always ends with

```
 APP    lager.app.src
make[1]: Leaving directory `/home/jenkins/workspace/myapp/deps/lager'
make[1]: Entering directory `/home/jenkins/workspace/myapp/deps/jiffy'
make[1]: *** No rule to make target `deps', needed by `rebar_dep'.  Stop.
make[1]: Leaving directory `/home/jenkins/workspace/myapp/deps/jiffy'
make: *** [deps] Error 2
```

erlang 19.3

**Comment by essen** [comment]
Oh yes I fixed that today I guess I'll push now. It does not always fail that's why I failed to notice it until now. Fixed in 7799804277e6fd38d308ebd581d194c38cd3fbb4. Update again with `make erlang-mk`. Enjoy!

**Comment by alexshavelev** [comment]
have made 4 build. all success
